src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -148,13 +145,9 @@ public Errors validatePartialUpdateDetails(OverseasEntitySubmissionDto overseasE
         var entityDto = overseasEntitySubmissionDto.getEntity();
 
         if (Objects.nonNull(entityDto)) {
-            var entityEmail = entityDto.getEmail();
-            // Temporary as initial public data Entity fetch has no Email Address. UAR-711
-            if (StringUtils.isNotBlank(entityEmail)) {
-                entityDtoValidator.validate(entityDto, errors, loggingContext);
-            }
+            entityDtoValidator.validate(entityDto, errors, loggingContext);
```

Comment:
This may cause problems on cidev where we don't have the oe email address.

Reply:
suggest removing from partial validation and only doing entity validation in full validation at end.